Sources/Run/Releaser.swift

```diff
@@ -10,19 +10,25 @@ struct Releaser {
         self.verbose = verbose
     }
     
-    func makeRelease(sha: String, tagOnly: Bool = false) async throws -> Version? {
+    func makeRelease(sha: String, tagOnly: Bool = false, suffix: String? = nil) async throws -> Version? {
         let (initialVersion, commits) = try await fetchCommits(sha: sha)
         let newVersion = try incrementVersion(initialVersion, commits: commits)
         
+        var newVersionDescription = newVersion.description
+        
+        if let suffix {
+            newVersionDescription += "-\(suffix)"
+        }
```

Comment:
I think we can avoid the mutability here 🙂 
```suggestion
        let newVersionDescription = if let suffix {
            "\(newVersion.description)-\(suffix)"
        } else {
            newVersion.description
        }
```